source/tasks/Utils/environment.ts

```diff
@@ -49,7 +49,7 @@ export const getVstsEnvironmentVariables= () : VstsEnvironmentVariables =>{
 
 export const generateReleaseNotesContent = (environment: VstsEnvironmentVariables, linkedItemReleaseNote: string, customReleaseNotes: string) => {
     let notes: string = "Release created by ";
-    const buildUri = `${environment.teamCollectionUri}${environment.projectName}/_build/index?_a=summary&buildId=${environment.buildId}`;
+    const buildUri = `${environment.teamCollectionUri}${encoreURIComponent(environment.projectName)}/_build/index?_a=summary&buildId=${encodeURIComponent(environment.buildId)}`;
```

Comment:
`encoreURIComponent` should be `encodeURIComponent` and we don't need to encode the `buildId` as it is typed as a number.